examples/traits/logging/README.md

```diff
@@ -0,0 +1,50 @@
+# Logging Trait
+
+This type of trait tells us what is going on in a running integration pod. It does this by printing logs out to the standard output.
+
+Logs are enabled by default on an integration. Just run this example:
+
+     $kamel run logging.java
```

Comment:
either remove the `$` or add a space to separate it from `kamel`. Same for the rest.